plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -878,7 +878,7 @@ private List<Document> guessTableFields(String schemaName, String tableName)
     {
         MongoDatabase db = client.getDatabase(schemaName);
         Document doc = db.getCollection(tableName).find().first();
-        if (doc == null) {
+        if (doc == null || doc.isEmpty()) {
```

Comment:
Please add a test to TestMongoConnectorTest.

Reply:
Sure will do.